Events/docs (#9100)
hashicorp/nomad: website/pages/docs/configuration/server.mdx

  this server will handle. This can be used to restrict the evaluations that
  worker threads will dequeue for processing.

> - `enable_event_broker` `(bool: true)` - Specifies if this server's state store

Reviewer: ```suggestion
- `enable_event_broker` `(bool: true)` - Specifies if this server
```

Mentioning the state store seems like an implementation detail - could we just say the server is doing this?

  [encryption documentation][encryption] for more details on this option
  and its impact on the cluster.

> - `event_buffer_size` `(int: 100)` - Specifies the amount of events generated

Reviewer: ```suggestion
- `event_buffer_size` `(int: 100)` - Specifies the number of events generated
```

Reviewer: Should we comment on why one might increase or decrease this value?

Author: yeah, how about 
```
- `event_buffer_size` `(int: 100)` - Specifies the number of events generated
  by the server's state store to be held in memory. Increasing this value
  enables new subscribers to have a larger lookback window when initially
  subscribing. Decreasing will lower the amount of memory used for the event
  buffer.
```

>   by the server's state store to be held in memory. 

Reviewer: ```suggestion
  by the server to be held in memory. 
```

likewise

  deployment must be in the terminal state before it is eligible for garbage
  collection. This is specified using a label suffix like "30s" or "1h".

> - `durable_event_count` `(int: 100)` - Specifies the amount of events to

Reviewer: ```suggestion
- `durable_event_count` `(int: 100)` - Specifies the number of events to
```

>   persist during snapshot generation. This provides a given servers event

Reviewer: ```suggestion
  persist during snapshot generation. This provides a given server's event
```

  stream durability during restarts by persiting a backlog of events to be

>   repopulated when the server comes back online.

Reviewer: Should we comment on why one might increase or decrease this value?
